package.json

```diff
@@ -32,11 +33,21 @@
     "@stardazed/streams": "^3.1.0",
     "@types/marked": "^2.0.3",
     "buffer": "^6.0.3",
+    "@walletconnect/client": "^1.4.2-rc.2",
+    "@walletconnect/qrcode-modal": "^1.4.1",
+    "@ethersproject/abstract-provider": "^5.4.0",
+    "@walletconnect/signer-connection": "^1.4.2-rc.2",
+    "@walletconnect/types": "^1.4.2-rc.2",
+    "@walletconnect/utils": "^1.4.2-rc.2",
     "ethereum-blockies": "^0.1.1",
+    "eventemitter3": "4.0.7",
+    "events": "^3.3.0",
```

Comment:
Do we need both event modules? Surely we should be able to get away with only one?